05/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Devmentor - 5 - practice html and css rwd</title>
+  </head>
+  <body>
+    <main class="wrapper">
+      <header>
+        <nav class="navigation">
+          <h1 class="logo">LOGO</h1>
+          <button class="hamburger"></button>
+          <ul class="main__menu">
+            <li><a href="#">Home</a></li>
+            <li><a href="#hero">Hero</a></li>
+            <li><a href="#about">About</a></li>
+          </ul>
+        </nav>
+      </header>
+      <article>
+        <section id="hero">
+          <div>
```

Comment:
To będzie wynikać z treści (której nie ma), ale bardziej się skłaniam ku temu, że tutaj właśnie będzie `<article>`, a nie na zewnątrz.